lms/djangoapps/teams/views.py

```diff
@@ -136,6 +137,15 @@ def get(self, request, course_id):
         topics = get_alphabetical_topics(course)
         organization_protection_status = user_organization_protection_status(request.user, course_key)
 
+        # We have some frontend logic that needs to know if we have any open, public, or managed teamsets,
+        # and it's easier to just figure that out here when we have them all already
+        has_open_teamset = any(topic for topic in topics if topic['type'] == TeamsetType.open.value)
+        has_public_managed_teamset = any(topic for topic in topics if topic['type'] == TeamsetType.public_managed.value)
+        has_private_managed_teamset = any(
```

Comment:
Same plural naming suggestion
```suggestion
        has_open_teamsets = any(topic for topic in topics if topic['type'] == TeamsetType.open.value)
        has_public_managed_teamsets = any(topic for topic in topics if topic['type'] == TeamsetType.public_managed.value)
        has_private_managed_teamsets = any(
```